packages/flutter_tools/lib/src/commands/build_bundle.dart

```diff
@@ -103,7 +103,7 @@ class BuildBundleCommand extends BuildSubCommand {
       case TargetPlatform.darwin_x64:
       case TargetPlatform.windows_x64:
       case TargetPlatform.linux_x64:
-        if (FlutterVersion.instance.isStable) {
+        if (!FlutterVersion.instance.isMaster) {
```

Comment:
Will this trigger for people who have renamed their master branch (e.g. folks working on this feature)?  I guess if it does we'll find out and can easily fix it.

Reply:
Also, another way to handle this would be to have a helper function isDesktopEnabled() and isWebEnabled() so that we can change this rule later w/o having to find all 10 spots again. :)